pyproject.toml

```diff
@@ -48,6 +48,7 @@ bandit = { version = "^1.7.4", extras = ["toml"] }
 liccheck = "^0.7.3"
 pylint = "2.16.4"
 pre-commit = "^3.1.1"
+pytest= "^7.2.2"
```

Comment:
Can't comment on the file above, but the PR adds an empty setup.cfg file in a mlpipelines/common folder, should not be the case